client/src/components/Main/Main.jsx

```diff
@@ -90,7 +89,6 @@ const Main = ({ history }) => {
         </Box>
       </Box>
       <Snackbar message={message} variant={variant} timeopened={timeOpened} />
-      )
```

Comment:
oh my GOD i've been searching for this bracket

Reply:
i thought it was somewhere in take survey and i couldn't find it

Reply:
sneaky